policyengine_us/variables/gov/states/va/tax/income/va_agi.py

```diff
@@ -0,0 +1,28 @@
+from policyengine_us.model_api import *
+
+
+class va_agi(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Virginia Adjusted Gross Income (VAGI)"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://law.lis.virginia.gov/vacodefull/title58.1/chapter3/article2/"
+    )
+    defined_for = StateCode.VA
+
+    adds = [
+        "adjusted_gross_income",
+        "va_additions",
+    ]
+    subtracts = [
+        "va_military_basic_pay_subtraction",
```

Comment:
make a va_subtractions.py and a list parameter that includes all subtarctions

Reply:
also need us_govt_interest, from point 1 (law) if I am not wrong

Reply:
double check all [subtarctions](https://law.lis.virginia.gov/vacodefull/title58.1/chapter3/article2/)